pangram.py

```diff
@@ -0,0 +1,40 @@
+# Solution 3 - O^n2 Quadratic Time
+
+# Question:
+# Each time I run the timer, it gives me a different result for the same problem. That makes sense because my computer is probably running different things at different times, but how can I use the timer as an accurate way to compare different solutions to each other?
```

Comment:
Generally speaking, the times should be relatively close, and you can take in average. In a problem like this, the substantive part of the program is going to run _so fast_ that irrelevant time spent on other things is going to mislead. In this case I'd focus just on the asymptotic analysis, rather than trying to time it :)